addons/test_mail/tests/test_mail_composer.py

```diff
@@ -1063,10 +1063,6 @@ def test_mail_composer_recipients(self):
                 })
 
                 # creation values are taken
-                if composition_mode == 'comment' and not batch_mode:
```

Comment:
Why removing it ? We should ensure it is correctly computed. Another reason to remove JS and use standard code.